### PR TITLE
#166800381 Ensure the error objects respect the structure provided in the README 

### PR DESCRIPTION
**Pull request by nkpremices**
##### What does this PR do?
Have the error objects updated to the right format 

##### Description of Task to be completed?
* Update the errors helper to the required format
* Use it at all the places it's needed

##### How should this be manually tested?
After cloning the repo, install all the dependencies then test the ```signup``` endpoint by providing invalid data.

### CODE REVIEW

**Review by Ramadhan0 — APPROVED** [review]

**Review by MCFrank16 — APPROVED** [review]
You are good man Genie Congolais

File: src/helpers/error.sender.js [hunk 1]
```diff
@@ -2,17 +2,18 @@
  * A function to create an error object and use the the
  * res object to send it back to the client
  * @param {Number} status - The status code to send back
- * @param {object} resultObj - The result object to structure
- * before to send back
  * @param {Object} res - The res object
+ * @param {String} fileld - the invalid filed rasing the error
```
Comment (MemunaHaruna) [inline src/helpers/error.sender.js]: @nkpremices do you mean `field - the invalid field raising the error`?

Also, check other instances of `fileld` and correct spelling.

**Review by MemunaHaruna — APPROVED** [review]

File: src/helpers/error.sender.js (hunk 1 above)
Comment (premlogic) [reply src/helpers/error.sender.js]: Oh no!! I've been avoiding this error hardly.
It won't happen again. 
I promise.

Comment (premlogic) [reply src/helpers/error.sender.js]: Let me correct it then check other instances

**Pull request merged**